docs/howto/account-billing/change-billing-data.md

```diff
@@ -1,49 +1,85 @@
-# Changing your account billing data
+# Changing your account data
 
 You may at any time change the contact person, address, company name, and purchase order number associated with your {{brand}} account.
 
 To get started, navigate to the [{{gui}}](https://{{gui_domain}}).
 Log in and click the _Profile_ button at the top right.
 
-![Orange avatar person with white background.](assets/picture-profile.png)
+![Orange avatar person with white background.](assets/profile.png)
 
 There, you will see the _Account settings_ page.
 Choose the _Customer Info_ tab, in which you can change and manage your customer information.
 
-![The "Customer Info" tab with a green "Update" button to complete the changes](assets/picture-changecustomerinfo2.png)
+![The 'Customer Info' tab with a green 'Update' button to complete the changes](assets/changecustomerinfo2.png)
 
 Finalize your changes by clicking the green _Update_ button at the bottom.
 
-## Changing an account's organization number
+## Adding or removing email addresses
 
-Please note that you cannot readily change your organization number (for business accounts) or your personal number (for individual accounts) in the {{gui}}.
+In the _Account settings_ pane, click the middle tab, labeled _Contact_.
+This will show your currently configured email address (or addresses).
+If there is only one email address, it will have all available roles activated.
```

Comment:
And while we are avoiding the parentheses, we can also be less repetitive:

```suggestion
If there is only one, it will have all available roles activated.
```

Reply:
Nope, because if I follow your prior suggestion, then the preceding sentence only has "addresses" in the plural, making the back-reference "only one" awkward-sounding. :)